src/tools/PDB.cpp

```diff
@@ -515,6 +528,12 @@ bool PDB::readFromFilepointer(FILE *fp,bool naturalUnits,double scale) {
     string z=line.substr(46,8);
     string occ=line.substr(54,6);
     string bet=line.substr(60,6);
+    string BoxX=line.substr(7,8);
```

Comment:
@SilvioPipolo I think these are incorrect. Please check [PDB doc](http://www.wwpdb.org/documentation/file-format-content/format33/sect8.html#CRYST1). I think it should be `string BoxX=line.substr(6,9);` and `string BoxA=line.substr(33,7);` etc